src/mainboard/clevo/tgl-u/variants/ns50mu/overridetree.cb

```diff
@@ -150,6 +150,14 @@ chip soc/intel/tigerlake
 				register "hid_desc_reg_offset" = "0x01"
 				device i2c 15 on end
 			end
+			chip drivers/i2c/hid
+				register "generic.hid" = ""FTCS1000""
+				register "generic.desc" = ""FocalTech Touchpad""
+				register "generic.irq_gpio" = "ACPI_GPIO_IRQ_LEVEL_LOW(GPP_B3)"
+				register "generic.probed" = "1"
```

Comment:
Please remove, it adds DSD with options only available for Linux which does not do what you think.

Reply:
Removed